Tests/PublishTests/Tests/ProjectGeneratorTests.swift

```diff
@@ -0,0 +1,43 @@
+/**
+*  Publish
+*  Copyright (c) John Sundell 2019
+*  MIT license, see LICENSE file for details
+*/
+
+import XCTest
+@testable import PublishCLICore
```

Comment:
I prefer not using `@testable import` within frameworks, since it often leads to testing too many implementation details, rather than the public API. There's already a test case called `CLITests` where you can perform end-to-end testing of your new functionality - which I think is the proper way to test CLI tools, that they actually produce the right outcome.

Reply:
I moved all the tests related to siteName to CLITests and got rid of `@testable import`.